.env.example

```diff
@@ -11,7 +11,7 @@ DB_PASSWORD=secret
 
 CACHE_DRIVER=file
 SESSION_DRIVER=file
-QUEUE_DRIVER=sync
+QUEUE_DRIVER=database
```

Comment:
I think we should keep sync by default, and not provide the database queue migration files.